docs/en_US/Tuner/InstallCustomizedTuner.md

```diff
@@ -28,16 +25,22 @@ This command build a whl file which is a pip installation source.
 
 Step 2: Run command:
 
-`nnictl package install dist/demo_tuner-0.1-py3-none-any.whl`
+`pip install dist/demo_tuner-0.1-py3-none-any.whl`
+
+## Register the customized tuner as builtin tuner:
+
+Run following command:
+
+`nnictl algo register --meta meta_file.yml`
 
-## Check the installed package
+## Check the registered builtin algorithms
 
-Then run command `nnictl package list`, you should be able to see that demotuner is installed:
+Then run command `nnictl algo list`, you should be able to see that demotuner is installed:
```

Comment:
do we still support `nnictl package` command?

Reply:
does it list nni built-in tuners?

Reply:
an error message is added to tell that `nnictl package` is replaced by `nnictl algo` while user using `nnictl package`